admin-dev/themes/new-theme/js/pages/product/components/dropzone/DropzoneWindow.vue

```diff
@@ -75,96 +75,98 @@
     >
-      <div
-        class="md-checkbox dropzone-window-checkbox"
-        v-if="selectedFile !== null"
-        :data-toggle="showCoverTooltip"
-        :data-original-title="$t('window.cantDisableCover')"
-      >
-        <label>
-          <input
-            type="checkbox"
-            :disabled="isCover"
-            :checked="isCover"
-            @change="$emit('coverChanged', $event)"
-          >
-          <i class="md-checkbox-control" />
-          {{ $t("window.useAsCover") }}
-        </label>
-      </div>
-
-      <input
-        type="file"
-        class="dropzone-window-filemanager"
-        @change="watchFiles"
-      >
-
-      <div class="dropzone-window-label">
-        <label
-          for="caption-textarea"
-          class="control-label"
-        >{{
-          $t("window.caption")
-        }}</label>
-        <div class="dropdown">
-          <button
-            class="btn btn-outline-secondary btn-sm dropdown-toggle js-locale-btn"
-            type="button"
-            data-toggle="dropdown"
-            aria-haspopup="true"
-            aria-expanded="false"
-            id="form_invoice_prefix"
-          >
-            {{ selectedLocale.iso_code }}
-          </button>
-          <div
-            class="dropdown-menu locale-dropdown-menu"
-            aria-labelledby="form_invoice_prefix"
-          >
-            <span
-              v-for="locale in locales"
-              :key="locale.name"
-              class="dropdown-item js-locale-item"
-              :data-locale="locale.iso_code"
-            >
-              {{ locale.name }}
-            </span>
-          </div>
-        </div>
-      </div>
+      <label>
+        <input
+          type="checkbox"
+          :disabled="isCover"
+          :checked="isCover"
+          @change="coverChanged"
+        >
+        <i class="md-checkbox-control" />
+        {{ $t('window.useAsCover') }}
+      </label>
+    </div>
 
-      <textarea
-        id="caption-textarea"
-        name="caption-textarea"
-        class="form-control"
-        v-model="selectedCaption"
-      />
+    <input
+      type="file"
+      class="dropzone-window-filemanager"
+      @change="watchFiles"
+    >
 
-      <div class="dropzone-window-button-container">
+    <div
+      class="dropzone-window-label"
+      v-if="selectedFile !== null"
+    >
+      <label
+        for="caption-textarea"
+        class="control-label"
+      >{{
+        $t('window.caption')
+      }}</label>
+      <div
+        class="dropdown"
+        v-if="locales.length > 1"
+      >
         <button
+          class="btn btn-outline-secondary btn-sm dropdown-toggle js-locale-btn"
           type="button"
-          class="btn btn-primary save-image-settings"
-          @click="$emit('saveSelectedFile')"
+          data-toggle="dropdown"
+          aria-haspopup="true"
+          aria-expanded="false"
+          id="form_invoice_prefix"
```

Comment:
Not sure I understand what invoice comes doing here? Maybe a copy-paste from the order page or something?